src/neo/Ledger/Blockchain.cs

```diff
@@ -422,7 +422,7 @@ private void OnTransaction(Transaction tx, bool relay)
             if (ContainsTransaction(tx.Hash))
                 SendRelayResult(tx, VerifyResult.AlreadyExists);
             else
-                txrouter.Tell(new TransactionRouter.Task { Transaction = tx, Relay = relay }, Sender);
+                txrouter.Tell(new TransactionRouter.Task { Transaction = tx, Relay = relay, Snapshot = GetSnapshot() }, Sender);
```

Comment:
Have you test TPS drop for this?? And, create one `SnapshotView` for each iincoming transaction????

Reply:
> And, create one SnapshotView for each iincoming transaction????

Fixed.